pipeline/tasks/protoc_tasks.py

```diff
@@ -262,6 +266,15 @@ def _prepare_pkg_dir(output_dir, api_name, api_version, organization_name,
         stderr=subprocess.STDOUT)
     return pkg_dir
 
+def _check_protoc_version(language):
```

Comment:
I don't think validating what version of protoc should be the business of an Artman task; we build the Docker images for situations where you want to enforce consistent installation of dependencies. (And it's brittle to validate by checking against a hardcoded string that we expect the protoc binary to print to stdout.)